versioned/gc/iceberg-actions/pom.xml

```diff
@@ -142,11 +146,6 @@
       <artifactId>DynamoDBLocal</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
we want it to be provided by the spark environment and it was at the bottom because of a dependency issue in jackson.